govcd/vapp.go

```diff
@@ -786,6 +798,329 @@ func (vapp *VApp) AddIsolatedNetwork(newIsolatedNetworkSettings *VappNetworkSett
 
 }
 
+// AddNetwork creates isolated or nat routed(connected to Org VDC network) network for vApp.
```

Comment:
This function (and the related `Async` function) need a better name.
In both `AddNetwork` and `AddOrgNetwork` we are passing an `OrgVdcNetwork` as a parameter, but in one case we are assigning an existing network to the vApp, and in the other we are creating a vApp network. 
I think the name in this case should be `CreateVappNetwork` or `AddVappNetwork` to avoid confusion

Reply:
@Didainius what is your vote for naming?

Reply:
renamed